README.md

```diff
@@ -39,13 +38,7 @@ Unless your just testing out MOA, we recommend using a dedicated AWS account to
 
 If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the [Red Hat Requirements for Customer Cloud Subscriptions](https://www.openshift.com/dedicated/ccs#scp) for details on the minimum required SCP.
 
-As part of the cluster creation process, `moactl` will perform the following actions:
-
-- Create an osdCcsAdmin IAM user:
-  - This user will have Programmatic access enabled.
-  - This user will have the AdministratorAccess policy attached to it.
-  - This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
-
+As part of the cluster creation process, `moactl` will create an osdCcsAdmin IAM user. This user will have Programmatic access enabled and have the AdministratorAccess policy attached to it. The user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
```

Comment:
Once the user is created its no longer leveraging the provided credentials. The last sentence (to me :smile: ) could be changed to something like "The provided AWS credentials in the next section will be used to create this user."